### DISCUSSION TITLE
None of saved filters show in list anymore

### DISCUSSION THREAD

**Discussion by HawkAPFM** [EmilyRosina/gsf-docs#6] [Feedback: Bug]
After the latest release none of my filters show in dropdown anymore. Oddly if I put the same query text in the box it recognizes it and renames to a saved filter name.  I can no longer use this as is.

**Comment by EmilyRosina** [comment]
Hello 👋 

Thanks for your feedback.
I found that happened to me a few versions ago and sometimes still when the page loads sometimes. For me a refresh fixes it and i'm able to see all my filters again.

Can you tell me a few things please?

- What Version you're using.
- What OS and browser you're using.
- If a refresh fixes it for you.
- What happens when you use the `beta` [backup](https://gsf-docs.4uroraskye.com/elements/filters-menu/additional-actions.html#backup-filters) feature; are all your filters saved to the Gist?

Cheers 😎
